### DISCUSSION TITLE
How many deploys can I do on Github Pages?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#65262] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi,

I would like to know how many deploys I can do on Github. How many deploys can I do on Github Pages? Are there any deployment limits on GitHub Pages? 

Context: For example, I have 200 html, css, js projects. Can I deploy these 200 html, css, js projects via GitHub Pages?

**Comment by dushmanta05** [comment]
There are no deployment limits on GitHub pages in terms of number, although you need to consider the following limitations while deploying:
- Only **static websites** can be deployed on GitHub pages.
- Published on GitHub Page sites may be **no larger than 1 GB**, which means all HTML, CSS, and JavaScript files should not be larger than 1GB (for a single repo).
- GitHub Pages sites have a **soft limit** of **10 builds per hour**, which means that for a single repo per hour, 10 builds can be triggered. Additional requests will be queued for the next hour.

For more details read the [**docs**](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#usage-limits).

**Reply by ghost** [reply]
Hi @dushmanta05, how are you?

thank you for all information here. Thanks for that very quick response, I was worried about the amount of forks I've been deploying.

**Reply by dushmanta05** [reply]
Hey @solarsbeans I'm doing well. How are you?
No worries; I'm glad my response helped.

**Reply by ghost** [reply]
Hi @dushmanta05, I'm fine too.

> No worries; I'm glad my response helped.

thanks!

**Reply by dushmanta05** [reply]
Absolutely, feel free to fork! There's a reason why it's public. Open-source is wonderful, as it promotes collaboration and brings industry-standard code to everyone.

**Reply by ghost** [reply]
Hi @dushmanta05 again.

> Absolutely, feel free to fork! There's a reason why it's public. Open-source is wonderful, as it promotes collaboration and brings industry-standard code to everyone.

thank you for feedback again. an question: Would you like to see a project I'm creating right now? ( If you want and have time I will send you the link of the github pages I created. if so send me a comment confirming)

**Reply by dushmanta05** [reply]
Definitely! Do share it.

**Reply by ghost** [reply]
Hi @dushmanta05.

> Definitely! Do share it.

I'm creating a comparison of open source, closed, exclusive licenses. It's like a benchmarking tool, but applied in software development. Please, see this: `https://solarsbeans.github.io/choose-all-license/`. I will add more licenses soon, the code is licensed under 'the unlicensed'.  The source code here: `https://github.com/solarsbeans/choose-all-license`

What do you think of this idea?

**Reply by dushmanta05** [reply]
Yeah, I saw. So basically, you're creating a comparison between different open source licenses so that developers can choose which license to use depending on their requirements, right? That's great.

**Reply by ghost** [reply]
Hi @dushmanta05.

> Yeah, I saw. So basically, you're creating a comparison between different open source licenses so that developers can choose which license to use depending on their requirements, right? That's great.

thanks again. yes, exactly!

**Reply by dushmanta05** [reply]
Hey, I can collaborate on your repo if you would like for the front-end part. I've experience using HTML, CSS, and JS.

**Reply by ghost** [reply]
Hi @dushmanta05 

> Hey, I can collaborate on your repo if you would like for the front-end part. I've experience using HTML, CSS, and JS.

It would be amazing to have help ^^.

**Reply by dushmanta05** [reply]
I see you have Twitter, I'll DM you there and let you know.

**Reply by ghost** [reply]
Hi @dushmanta05,

> I see you have Twitter, I'll DM you there and let you know.

thanks!
